cmake/CMakeLists.txt

```diff
@@ -80,6 +80,7 @@ option(onnxruntime_USE_EIGEN_THREADPOOL "Use eigen threadpool. Otherwise OpenMP
 option(tensorflow_C_PACKAGE_PATH "Path to tensorflow C package installation dir")
 option(onnxruntime_ENABLE_LANGUAGE_INTEROP_OPS "Enable operator implemented in language other than cpp" OFF)
 option(onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS "Dump node input shapes and output data to standard output when executing the model." OFF)
+option(onnxruntime_BUILD_JNI "Build with JNI" OFF)
```

Comment:
Or we could check if java is available in to PATH, if it is, just enable it. I feel that would be simpler for end users.

Reply:
This is likely to cause issues if there are multiple Java versions on the PATH. Especially if it's targetting Android as Android only supports Java 8, and the currently supported Java release is Java 12.